errors.js

```diff
@@ -648,6 +648,10 @@ Errors.classify = function classify(err) {
         return err.codeName;
     }
 
+    if (err.type.indexOf('bufrw.') > -1) {
+        return 'NetworkError';
```

Comment:
Yeah, in particular what we're seeing is a bufrw short error in connection closing.  Seems like remotes side is closing mid-frame-write.